pkg/cmd/root.go

```diff
@@ -71,3 +71,10 @@ func errorIfFileDoesNotExist(val *types.CmdValues) {
 		os.Exit(1)
 	}
 }
+
+func setDefaultComposeFile(val *types.CmdValues) {
+	val.Files = []string{"docker-compose.yml"}
+	if utils.CheckIfFileExists(val.Files) != nil {
+		val.Files = []string{"docker-compose.yaml"}
```

Comment:
Or do one thing like also check if `docker-compose.yaml` exists and if it does not give a generic error like docker-compose files not found, that would remove this from being showing docker-compose.yaml does not exists.